Application/src/visuals/Draw.java

```diff
@@ -39,6 +32,14 @@ Canvas getCanvas() {
         name = n;
     }
 
+    public String getName() {
```

Comment:
Add comments for Javadoc? or did we decide not to bother with the getters/setters?

Reply:
I didn't want to change the getters/setters in Draw because it predated the standard and many branches use it currently.